app/Controllers/Http/ProfilesController.ts

```diff
@@ -0,0 +1,65 @@
+// import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
+import Profile from 'App/Models/Profile';
+import { schema, rules } from '@ioc:Adonis/Core/Validator';
+export default class ProfilesController {
+    public async addProfile({ auth, request, response }) {
+        const newUserSchema = schema.create({
+            name: schema.string([
+                rules.minLength(3),
+                rules.maxLength(30),
+            ]),
+            email: schema.string([
+                rules.email()
+            ]),
+            gender: schema.string([]),
+            dob: schema.date.optional({
+                format: 'dd-MM-yyyy',
+            }),
+            user_id: auth.user.toJSON().id,
```

Comment:
Remove user_id from schema since we are not validating it.

Pass it in payload instead.

Reply:
Also, we can use `auth.user` directly, no need to use `toJSON`